### ISSUE TITLE
`accessibilityHint` appears to be unsupported - is this intentional?

### ISSUE THREAD

**Issue by AlanSl** [necolas/react-native-web#2029]
**The problem**

In this issue "New accessibility props" https://github.com/necolas/react-native-web/issues/1112 an intention is given to support React Native's `accessibilityHint` props on web:

> Implement accessibilityHint as it is done on Android: append the hint text to the accessibilityLabel. Consider exposing an option (e.g., on AccessibilityInfo) to disable hints on web, as unlike iOS (and like Android) there is no system-level option to disable hints.

Sounds good; however, this appears to not be implemented, and that issue is the only match for `accessibilityHint` that comes up in a repo-wide search:

![screenshot of search results for a search on "accessibilityHint" in the react-native-web repository, showing that the only match is the above-linked issue](https://user-images.githubusercontent.com/29628323/119121632-8dc53500-ba25-11eb-90b5-61e624461cd5.png)

It'd be good to know if, either, this is a missing feature that is still planned to be implemented, or if a good reason was found to not implement `accessibilityHint` support on web and a decision has been made to not support it.

**How to reproduce**

Simplified test case: https://codesandbox.io/s/wispy-surf-9c1t4?file=/src/App.js

Steps to reproduce:
1. Add an `accessibilityHint` to an element
2. Look in the HTML markup for it, it's not there

**Expected behavior**

It should either be supported, or on the roadmap to be supported, or, there should be something (ideally in the in the [accessibility docs](https://necolas.github.io/react-native-web/docs/accessibility/#:~:text=Accessibility%20in%20React%20Native%20for,attributes%2C%20and%20ARIA%20in%20HTML)) saying why the decision was made to not support it.

**Environment (include versions). Did this work in previous versions?**

* React Native for Web (version): 0.14.13
* React (version): 16.13.1
* Browser: any

**Comment by necolas** [comment]
It's not supported and there are no plans to support it. There are issues with the Android way of trying to support it (e.g., internationalisation), and it's not needed on web. I don't think it needs to be documented because you can see there's no support by the lack of documentation
